src/pyfmi/fmi.pyx

```diff
@@ -607,21 +671,35 @@ cdef class ModelBase:
                     if (f != NULL):
                         FMIL.fprintf(f, "FMIL: module = %s, log level = %d: %s\n",c_module, log_level, c_message)
                         FMIL.fclose(f)
+        elif self._log_stream:
+            try:
+                self._log_stream.write(msg)
+            except:
```

Comment:
Maybe a clarifying comment about what we are capturing here?

Reply:
Fixed